tests/system/Commands/Translation/LocalizationFinderTest.php

```diff
@@ -0,0 +1,226 @@
+<?php
+
+/**
+ * This file is part of CodeIgniter 4 framework.
+ *
+ * (c) CodeIgniter Foundation <admin@codeigniter.com>
+ *
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+namespace CodeIgniter\Commands\Translation;
+
+use CodeIgniter\Test\CIUnitTestCase;
+use CodeIgniter\Test\StreamFilterTrait;
+use Config\App;
+use Config\Services;
+use Locale;
+
+/**
+ * @group Others
+ *
+ * @internal
+ */
+final class LocalizationFinderTest extends CIUnitTestCase
+{
+    use StreamFilterTrait;
+
+    private static string $locale;
+    private static string $languageTestPath;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+        self::$locale           = Locale::getDefault();
+        self::$languageTestPath = SUPPORTPATH . 'Language/';
+        $this->clearGeneratedFiles();
+    }
+
+    public function testUpdateDefaultLocale(): void
+    {
+        @mkdir(self::$languageTestPath . self::$locale, 0777, true);
+        Services::commands()->run('lang:find', [
+            'dir' => 'Translation',
+        ]);
+        $this->realizeAssertion();
+        $this->clearGeneratedFiles();
```

Comment:
Please add empty lines. Arrange - Act - Assert.

```suggestion
        @mkdir(self::$languageTestPath . self::$locale, 0777, true);

        Services::commands()->run('lang:find', [
            'dir' => 'Translation',
        ]);

        $this->realizeAssertion();
```